css/index.css

```
@@ -634,3 +634,189 @@ body {
   }
 }
 
+/* Estilo para móvil */
+.functionality {
+  display: flex;
+  flex-direction: column;
+  background: #FFFFFF;
+  justify-content: center;
+  align-items: center;
+  padding: 20px;
+  position: relative;
+}
+
+.functionality__information {
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  width: 100%;
+  overflow: hidden;
+}
+
+.functionality__carousel-wrapper {
+  overflow: hidden;
+  width: 100%;
+}
+
+.functionality__carousel {
+  display: flex;
+  transition: transform 0.3s ease;
+}
+
+.functionality__icons {
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  border-radius: 10px;
+  overflow: hidden;
+  background-color: white;
+  gap: 40px;
+  min-width: 100%;
+  box-sizing: border-box;
+  padding: 20px;
+}
+
+.functionality__fund {
+  position: relative;
+  overflow: hidden;
+  width: 86.59px;
+  height: 103.72px;
+  border-radius: 12px;
+  flex-shrink: 0;
+}
+
+.functionality__overlay {
+  position: absolute;
+  top: 0;
+  left: 0;
+  width: 100%;
+  height: 100%;
+  background-color: #FFCCB4;
+  opacity: 0.2;
+  border-radius: 12px;
+  z-index: 1;
+}
+
+.functionality__img {
+  width: 53.29px;
+  height: 51.22px;
+  position: absolute;
+  top: 50%;
+  left: 50%;
+  transform: translate(-50%, -50%);
+  z-index: 2;
+}
+
+.functionality__text {
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  gap: 15px;
+}
+
+.functionality__paragraph {
+  font-size: 20px;
+  font-weight: 600;
+  line-height: 30px;
+  letter-spacing: 1%;
+  text-align: center;
+  color: #363535;
+}
+
+.functionality__Saber-mas {
+  font-size: 14px;
+  font-weight: 600;
+  line-height: 20px;
+  letter-spacing: -1%;
+  text-align: center;
+  color: #D14D41;
+}
+
+.carousel__controls {
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+  width: 60%;
+  position: absolute;
+  transform: translateY(-50%);
+}
+
+.carousel__prev,
+.carousel__next {
+  background: none;
+  border: none;
+  font-size: 40px;
+  cursor: pointer;
+  color: #D14D41;
+  opacity: 0.2;
+}
+
+.carousel__indicators {
+  position: absolute;
+  bottom: -30px;
+  display: flex;
+  justify-content: center;
+  width: 100%;
+}
+
+.carousel__indicator {
+  width: 10px;
+  height: 10px;
+  background: #ddd;
+  border-radius: 50%;
+  margin: 0 5px;
+  cursor: pointer;
+  background-color: #fadfdc;
+  border: none;
+}
+
+.carousel-indicator--active {
```

Posible mala escritura aqui. No tienes alguna clase definida que se llame `carousel-indicator`
```suggestion
.carousel__indicator--active {
```